### PR TITLE
GH action for installing / uninstalling UI versions

### PR DESCRIPTION
**Pull request by oliver-sanders**
Dead simple action to codify the UI update process.

closes https://github.com/cylc/cylc-uiserver/issues/187

example run https://github.com/oliver-sanders/cylc-uiserver/pull/1 (note fixed the commit message)

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Does not need tests (why?).
- [x] No change log entry required (why? e.g. invisible to users).
- [x] No documentation update required.
- [x] No dependency changes.

### CODE REVIEW

**Review by MetRonnie — APPROVED** [review]
LGTM

File: .github/workflows/update_ui.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+jobs:
+  ui:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    env:
+      BRANCH: update-ui-${{github.run_number}}
+
+    steps:
+      - name: checkout cylc-doc
```
Comment (kinow) [inline .github/workflows/update_ui.yml:21]: Copied from cylc doc?

File: .github/workflows/update_ui.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      - name: install
+        env:
+          RELEASES: https://github.com/cylc/cylc-ui/releases/download
+        run: |
+          for version in ${{ github.event.inputs.install }}; do
+            wget \
+              "$RELEASES/${version}/cylc-ui-${version}-dist.zip" \
+              -O "${version}.zip"
+            mkdir -p "cylc/uiserver/ui/${version}/"
+            unzip 0.2.zip -d "cylc/uiserver/ui/${version}/"
```
Comment (kinow) [inline .github/workflows/update_ui.yml:47]: Shouldn't 0.2 zip be something like $version.zip?

**Review by kinow — CHANGES_REQUESTED** [review]

File: .github/workflows/update_ui.yml (hunk 2 above)
Comment (oliver-sanders) [reply .github/workflows/update_ui.yml]: 🤦 well you can tell which version I tested it with!

**Review by kinow — APPROVED** [review]

**Pull request merged**